src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1709,7 +1709,11 @@ public String getQuestionText(String feedbackQuestionId) {
         return sortedMap;
     }
 
-    public LinkedHashMap<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>>
+    /**
+     * Returns an ordered Map with {@code recipientTeam} name as key
+     * sorted by {@code compareByTeamQuestionRecipientTeamGiver} comparator.
```

Comment:
`compareByTeamQuestionRecipientTeamGiver` is a private variable. It doesn't make sense to explain a public method using a private variable. Perhaps explain the sorting order in plain English? @xpdavid what do you think?